homeassistant/components/homekit/__init__.py

```diff
@@ -214,6 +217,10 @@ def add_bridge_accessory(self, state):
         """Try adding accessory to bridge if configured beforehand."""
         if not state or not self._filter(state.entity_id):
             return
+        self.devices_added += 1
```

Comment:
You don't need a counter var for this. All accessories are saved in the `accessories` dict of the bridge. So
```py
len(self.bridge.accessories)
```
is enough.